File: Hook/HookSkinDemo/app/src/main/java/com/dream/hookskindemo/MyApplication.java
package com.dream.hookskindemo;

import android.app.Application;

import com.dream.hookskindemo.skin.SkinEngine;
import com.dream.hookskindemo.utils.FileUtils;

/**
 * function: waiting for add
 *
 * @author zy
 * @since 2022/11/13
 */
public class MyApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        FileUtils.copyAssetsFileToExternalCache(this,"skin.apk");


        SkinEngine.getInstance().init(this);
    }
}

File: PerformanceOpt/PerformanceOptimization/app/src/main/java/com/dream/performanceoptimization/MyApplication.kt
package com.dream.performanceoptimization

import android.app.Application
import android.content.Context

/**
 * function: waiting for add
 *
 * @author zy
 * @since 2023/3/27
 */
class MyApplication: Application() {

    override fun attachBaseContext(base: Context?) {
        super.attachBaseContext(base)
    }


    override fun onCreate() {
        super.onCreate()
    }
}